shacl12-ui/index.html

```diff
@@ -801,8 +801,636 @@ <h3>Document Conventions</h3>
 
     <section id="widgets">
       <h2>Widgets</h2>
-      <p>Content.</p>
-      <p class="todo">Remove this reference test: [[RDF12-PRIMER]]</p>
+
+      <section id="editors">
+        <h3>Editors</h3>
+        <p>
+          The following sub-sections enumerate the currently defined instances of <code>shui:Editor</code> from the SHACL UI namespace.
+          Property shapes can explicitly specify the preferred editor for its values using <code>shui:editor</code>.
+          If no such value has been specified, the system should pick a suitable default viewer based on the
+          <a href="#score">scoring system</a> outlined for each widget.
+        </p>
+        <section id="AutoCompleteEditor">
+          <h4>shui:AutoCompleteEditor</h4>
+          <p>
+            <b>Score:</b>
+            <ul>
+              <li>1 if the value is an IRI.</li>
+              <li>0 otherwise.</li>
+            </ul>
+          </p>
+          <p>
+            <b>Rendering:</b><br>
```

Comment:
```suggestion
            <b>Rendering:</b>
```
for consistency with the other subsections